src/styles/_catalog.scss

```diff
@@ -16,6 +16,12 @@
     margin-bottom: 48px;
   }
 
+  &__card:hover {
+    transform: scale(1.2);
+    color: #34568b;
+    transition-duration: 300ms;
+  }
```

Comment:
transition property should be in parent element (&__card) and you better specify on which property you use transition.